torch/_inductor/codegen/wrapper.py

```diff
@@ -213,53 +142,13 @@ def codegen(self, code: IndentedBuffer):
         assert self.node.get_name() not in V.graph.removed_buffers
         assert self.reused_as.get_name() not in V.graph.removed_buffers
         code.writeline(
-            make_buffer_reuse(
-                self.node,
-                self.reused_as,
-                del_func=lambda name: f"; del {name}",
-                declare="",
-                ending="",
-                as_strided="as_strided",
-            )
-            + "  # reuse"
-        )
-
-
-@dataclasses.dataclass
-class CppReuseLine(ReuseLine):
-    node: ir.Buffer
-    reused_as: ir.Buffer
-
-    def codegen(self, code: IndentedBuffer):
-        assert self.node.get_name() not in V.graph.removed_buffers
-        assert self.reused_as.get_name() not in V.graph.removed_buffers
-        code.writeline(
-            make_buffer_reuse(
+            self.wrapper.make_buffer_reuse(
                 self.node,
                 self.reused_as,
-                del_func=lambda name: f"; {name}.reset()",
-                declare="auto ",
-                ending=";",
-                as_strided="at::as_strided",
```

Comment:
do we no longer need to generate cpp `at::as_strided` calls?

Reply:
We still need `at::as_strided`, but now it is generated in make_buffer_reuse which works for both WrapperCodeGen and CppWrapperCodeGen.